src/plugins/proxyconfig_stem.py

```diff
@@ -28,15 +31,13 @@ def __call__(self, line):
             # Plugin's debug or unexpected log line from tor
             self._logger.debug(line)
         else:
-            self._logger.log(self._levels.get(level, 10), '(tor)' + line)
+            self._logger.log(self._levels.get(level, 10), '(tor)' + line)    # pylint: disable=logging-not-lazy
 
 
 def connect_plugin(config):
     """Run stem proxy configurator"""
     logwrite = DebugLogger()
-    if config.safeGet('bitmessagesettings', 'sockshostname') not in (
-        'localhost', '127.0.0.1', ''
-    ):
+    if config.safeGet('bitmessagesettings', 'sockshostname') not in ('localhost', '127.0.0.1', ''):
```

Comment:
len 99 - should be < 80

Reply:
On setup.cfg file, the max-line-length set to 119, that why i change the line